arrow/src/main/java/org/apache/iceberg/arrow/vectorized/parquet/VectorizedColumnIterator.java

```diff
@@ -278,10 +264,6 @@ public VarWidthTypeBatchReader varWidthTypeBatchReader() {
     return new VarWidthTypeBatchReader();
   }
 
-  public FixedWidthTypeBinaryBatchReader fixedWidthTypeBinaryBatchReader() {
-    return new FixedWidthTypeBinaryBatchReader();
-  }
```

Comment:
I saw in my IDE that this method is never called, which means that the chain of fixed width binary reader classes are never used.